### ISSUE TITLE
Event Reports available/accessible from Event Actions menu (or any other)

### ISSUE THREAD

**Issue by lu-chi** [MISP/MISP#7811]
Greetings! 
I guess it would be nice to have Event Reports accessible either from Event Actions or any other menu. For the moment it is available only by using URL (https://misp-server/eventReports/index) and from within particular event when choosing -> Event reports -> Actions only once report is generated.
IT would be great to be able to access all the generated reports at all time (might be also with access granted for users/groups). 
Thanks in advance!
